### PR TITLE
[cloud-provider-aws] Reduce cloud-init data size

### PR DESCRIPTION
**Pull request by vitaliy-sn**
## Description

The size of the cloud-init data has been reduced because we're limited to 16384 bytes

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-aws
type: fix
summary: The size of the cloud-init data has been reduced
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by vitaliy-sn** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 6299366839 6299366839-1-con-1-25 /sys/deckhouse-oss/install:pr5939 centos@3.79.217.11

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

File: modules/040-node-manager/templates/node-group/_bootstrap_script.tpl [hunk 1]
```diff
@@ -101,6 +101,9 @@ function detect_bundle() {
   {{- tpl ($context.Files.Get "candi/bashible/detect_bundle.sh") $context | nindent 2 }}
 }
 
+mkdir -p /opt/deckhouse/tmp
+export TMPDIR=/opt/deckhouse/tmp
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/templates/node-group/_bootstrap_script.tpl:105]:
```suggestion
export TMPDIR=/opt/deckhouse/tmp
mkdir -p "$TMPDIR"
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
A Fistful of Symbols

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Review by yalosev — APPROVED** [review]

**Pull request merged**